test/DynamoCoreWpfTests/CoreUITests.cs

```diff
@@ -786,7 +787,7 @@ private void RestartTestSetup(bool startInTestMode)
 
             //create the view
             View = new DynamoView(ViewModel);
-            SynchronizationContext.SetSynchronizationContext(new SynchronizationContext());
```

Comment:
these methods are only used for a few tests - correct?

Reply:
yes..just for 2 tests